### PR TITLE
DuckDNS uses https over http

### PR DESCRIPTION
**Pull request by psiegl**

### CODE REVIEW

File: ddclient [hunk 1]
```diff
@@ -4791,7 +4791,7 @@ Configuration variables applicable to the 'duckdns' protocol are:
 
 Example ${program}.conf file entries:
   ## single host update
-  protocol=duckdns,                                       \\
+  protocol=duckdns                                       \\
   password=z0mgs3cjur3p4ss                    \\
```
Comment (SuperSandro2000) [inline ddclient]:
```suggestion
  protocol=duckdns,                                      \\
  password=z0mgs3cjur3p4ss,                              \\
```

Comment (psiegl) [reply ddclient]: fixed

File: ddclient [hunk 2]
```diff
@@ -4816,7 +4816,7 @@ sub nic_duckdns_update {
 
         # Set the URL that we're going to to update
         my $url;
-        $url  = "http://$config{$h}{'server'}/update";
+        $url  = "https://$config{$h}{'server'}/update";
```
Comment (alivahabi2112) [inline ddclient]: bc1qex7a52dr8dz9dt7sn76t5fu9pdyzluwsdawapc

Comment (psiegl) [reply ddclient]: not sure what this means?!

File: ddclient [hunk 3]
```diff
@@ -4792,7 +4792,7 @@ Configuration variables applicable to the 'duckdns' protocol are:
 Example ${program}.conf file entries:
   ## single host update
   protocol=duckdns,                                       \\
-  password=z0mgs3cjur3p4ss                    \\
+  password=z0mgs3cjur3p4ss,                    \\
```
Comment (SuperSandro2000) [inline ddclient:4795]:
```suggestion
  password=z0mgs3cjur3p4ss,                           \\
```

Comment (SuperSandro2000) [inline ddclient:4795]:
```suggestion
  password=your_password,                                 \\
```

**Pull request merged**